File: vue-coffee-stuff/src/models/types/coffeeMachine.ts
export type EspressoMachine = {
    manufacturer: string,
    model: string,
    boilerType: BoilerType,
    price: number,
    year: number
}

export enum BoilerType {
    SingleBoiler = "single boiler",
    Thermocoil = "thermocoil",
    Thermoblock = "thermoblock",
    HeatExchanger = "heat exchanger",
    DualBoiler = "dual boiler",
}
  

File: dotnet-coffee-stuff/coffee-webapi/coffee-webapi/Models/CoffeeMachines/CoffeeMachine.cs
namespace coffee_webapi.Models.CoffeeMachines;

public class CoffeeMachine
{
    public int Id { get; set; }
    public string Manufacturer { get; set; }
    public string Model { get; set; }
    public int Year { get; set; }
    public decimal Price { get; set; }
}